docscan/session/create/check/liveness.go

```diff
@@ -36,34 +49,33 @@ func NewRequestedLivenessCheckBuilder() *RequestedLivenessCheckBuilder {
 type RequestedLivenessCheckBuilder struct {
 	livenessType string
 	maxRetries   *int
```

Comment:
Why is this a pointer?

Reply:
We want the default value to be nil, not 0

Reply:
Have changed to not be a pointer in 465ba75, with a test to check it's omitted from the payload if not specified